CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 - All the modules in [`pkg/sdk`](https://github.com/operator-framework/operator-sdk/tree/master/pkg/sdk) have been combined into a single package. `action`, `handler`, `informer` `types` and `query` pkgs have been consolidated into `pkg/sdk`. [#242](https://github.com/operator-framework/operator-sdk/pull/242)
 - The SDK exposes the Kubernetes clientset via `k8sclient.GetKubeClient()` #295
+- Logs generated by the informer in [`pkg/sdk`](https://github.com/operator-framework/operator-sdk/tree/master/pkg/sdk) have been refactored to have more appropriate logging levels. #309
```

Comment:
I think it's fine not to mention this change in the CHANGELOG. That's more for API/feature changes or bug fixes.